malariagen_data/af1.py

```diff
@@ -1,36 +1,14 @@
 import sys
 
-from .anopheles import AnophelesDataResource
-
-try:
-    # noinspection PyPackageRequirements
-    from google import colab
-except ImportError:
-    colab = None
-
 import malariagen_data  # used for .__version__
 
+from .anopheles import AnophelesDataResource
+
 MAJOR_VERSION_NUMBER = 1
 MAJOR_VERSION_PATH = "v1.0"
 CONFIG_PATH = "v1.0-config.json"
-
 GCS_URL = "gs://vo_afun_release/"
-
-GENOME_FASTA_PATH = (
-    "reference/genome/idAnoFuneDA-416_04/idAnoFuneDA-416_04_1.curated_primary.fa"
-)
-GENOME_FAI_PATH = (
-    "reference/genome/idAnoFuneDA-416_04/idAnoFuneDA-416_04_1.curated_primary.fa.fai"
-)
-GENOME_ZARR_PATH = (
-    "reference/genome/idAnoFuneDA-416_04/idAnoFuneDA-416_04_1.curated_primary.zarr"
-)
 SITE_ANNOTATIONS_ZARR_PATH = "reference/genome/idAnoFuneDA-416_04/Anopheles-funestus-DA-416_04_1_SEQANNOTATION.zarr"
-GENOME_REF_ID = "idAnoFuneDA-416_04"
-GENOME_REF_NAME = "Anopheles funestus"
-
-CONTIGS = "2RL", "3RL", "X"
```

Comment:
This should have been removed in an earlier PR, no longer needed because provided via the release config file.